litmus-portal/frontend/src/components/Sections/Settings/TeammingTab/Invitation/Sent/styles.ts

```diff
@@ -0,0 +1,33 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
+  table: {
+    maxHeight: '23.1875rem',
+  },
+  avatarBackground: {
+    backgroundColor: theme.palette.secondary.main,
+    width: '2.56rem',
+    height: '2.56rem',
+    color: '#FFF',
```

Comment:
we need to take every value from theme so it is easier for @chandankumar4 when he modifies it.
can you change these fixed values and take them from theme